test/e2e/serial/tests/workload_placement.go

```diff
@@ -413,6 +413,238 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 				return ok
 			}, time.Minute, time.Second*5).Should(BeTrue(), "resources not restored on %q", updatedPod.Spec.NodeName)
 		})
+		It("[test_id:48746][tier2] should modify workload post scheduling while keeping the resource requests available across all NUMA node", func() {
+			// NOTE: Initial plan for this test was to create a deployment with two replicas that are allocated resources from the two different NUMA nodes.
+			// However, since deployment controller tries to strictly spread its replicas(pods) across multiple nodes, there is no way to override
+			// this behaviour and place two replicas that are part of a deployment on the same node. Even after padding the second node,
+			// the second replica is left in a pending state waiting for resources to be made available on the second node. Because of this, this test
+			// is downsized and we create a deployment with single replica and two containers to achieve the same behaviour.
+
+			hostsRequired := 2
+			paddedNodes := padder.GetPaddedNodes()
+			paddedNodesSet := sets.NewString(paddedNodes...)
+
+			nrtInitialList, err := e2enrt.GetUpdated(fxt.Client, nrtv1alpha1.NodeResourceTopologyList{}, time.Second*10)
```

Comment:
minor: to list NRTs without any extra constraints, just use
```
err = fxt.Client.List(context.TODO(), &nrtList)
Expect(err).ToNot(HaveOccurred())
```
(yes, this means elsewhere in the code we do it wrongly)

Reply:
Ack, I will update.